lc0_main.go

```diff
@@ -140,6 +140,9 @@ func uploadGame(httpClient *http.Client, path string, pgn string,
 			continue
 		}
 		resp.Body.Close()
+		if strings.Contains(body.String(), "upgrade") {
```

Comment:
I would verify response.statuscode != 200 before doing this check - and maybe put spaces around the upgrade check to avoid it potentially matching some partial word (as unlikely as that would seem).

Reply:
Done